syntax = "proto3";

message LampadaControl {
    string control = 1;
}

message Lampada {
    string IP = 1;
    string tipo = 2;
    bool status = 3;
    bool ligada = 1;
}
